Modules/Devices/src/Devices.Application/DTOs/IdentitySummaryDTO.cs

```diff
@@ -20,6 +20,8 @@ public class IdentitySummaryDTO
 
     public IEnumerable<DeviceDTO> Devices { get; set; }
 
+    public IEnumerable<QuotaDTO> Quotas { get; set; }
```

Comment:
Wtf. Why does the Devices module now know what Quotas are? 😮 That's definitely not what we want! The modules have to stay strongly separated.

And even if it was correct to have it here: why do you add the Quotas to the SummaryDTO?!

Reply:
You last suggestion is close to what we need.

But instead of having only two properties, merge the properties of the two IdentityDTOs you receive from the two queries, so that it results in the following JSON response:

```json
{
  "address": "id1...",
  "createdAt": "2023-...."
  ...
  "quotas": [
    {
      "id": "...",
      "max": 5,
      ...
    }
  ]
}
```